### PR TITLE
Added wms/wfs schemas and a first version of the json2qgs application

### PR DESCRIPTION
**Pull request by HusseinKabbout**

### CODE REVIEW

**Comment by pka** [comment]
Schema:
* `composer` -> `print_template`

File: schemas/sogis-wfs-qgs-content.json [hunk 1]
```diff
@@ -27,21 +27,39 @@
...
+            "template_base64": {
               "description": "Print Composer template in Base64 encoding",
               "type": "string",
               "contentEncoding": "base64"
+            },
+            "template_assets": {
+              "title": "Assets list used by the template",
+              "type": "array",
+              "items": {
+                "title": "QTP assets",
```
Comment (pka) [inline schemas/sogis-wfs-qgs-content.json]: Typo? -> QPT

Comment (kabbohus) [reply schemas/sogis-wfs-qgs-content.json]: Thanks for pointing out the typo!

Comment (HusseinKabbout) [reply schemas/sogis-wfs-qgs-content.json]: Thanks for pointing out the typo!

**Review by mwa — APPROVED** [review]
Looks fine to me, thanks

**Pull request merged**